tests/base/test_os.lua

```diff
@@ -184,8 +184,26 @@
 			end
 		end
 	end
-
-
+	
+	-- Check outputof content
+	function suite.outputof_streams_output()
+		if (os.istarget("macosx")
+			or os.istarget("linux")
+			or os.istarget("solaris")
+			or os.istarget("bsd"))
+			and os.isdir (_TESTS_DIR)
+		then
+			local ob, e = os.outputof ("ls " .. _TESTS_DIR .. "/base")
+			local oo, e = os.outputof ("ls " .. _TESTS_DIR .. "/base", "output")
+			test.isequal (oo, ob)
+			local s, e = string.find (oo, "test_os.lua")
+			test.istrue(s ~= nil)
+			
+			local o, e = os.outputof ("ls " .. cwd .. "/base", "error")
+			test.istrue(o == nil or #o == 0)

```

Comment:
This test should probably validate that using `"error"` actually works, this just confirms that it doesn't get anything from the `"output"` stream.

Reply:
When I wrote this patch, I had more relevant tests that works locally but failed in the github actions context. So I added this minimal test. Note that the master branch does not contain any test about `os.outputof` output (only error code)